ebean-core/src/test/java/org/tests/batchload/TestBatchLazyWithCacheHits.java

```diff
@@ -17,66 +17,53 @@
 
 public class TestBatchLazyWithCacheHits extends BaseTestCase {
 
-  private ServerCache beanCache = server().getServerCacheManager().getBeanCache(UUOne.class);
+  private final ServerCache beanCache = server().getServerCacheManager().getBeanCache(UUOne.class);
 
   private UUOne insert(String name) {
     UUOne one = new UUOne();
     one.setName("testBLWCH" + name);
-    Ebean.save(one);
+    DB.save(one);
     return one;
   }
 
   @Test
   public void testOnCacheHit() {
-
-    ArrayList<UUOne> inserted = insertData();
-
+    List<UUOne> inserted = insertData();
     clearCacheAndStatistics();
 
-    UUOne b = Ebean.find(UUOne.class, inserted.get(1).getId());
-    assertNotNull(b);
-
-    UUOne b2 = Ebean.find(UUOne.class, inserted.get(1).getId());
-    assertNotNull(b2);
-
+    assertNotNull(DB.find(UUOne.class, inserted.get(1).getId()));
+    // cache hit
+    assertNotNull(DB.find(UUOne.class, inserted.get(1).getId()));
     assertBeanCacheHits(1);
 
-    UUOne c = Ebean.find(UUOne.class)
-      .where().idEq(inserted.get(2).getId())
-      .findOne();
-    assertNotNull(c);
-
-    UUOne c2 = Ebean.find(UUOne.class)
-      .where().idEq(inserted.get(2).getId())
-      .findOne();
-
-    assertNotNull(c2);
+    assertNotNull(DB.find(UUOne.class).where().idEq(inserted.get(2).getId()).findOne());
+    // cache hit
+    assertNotNull(DB.find(UUOne.class).where().idEq(inserted.get(2).getId()).findOne());
     assertBeanCacheHits(1);
 
     LoggedSqlCollector.start();
 
-    List<UUOne> list = Ebean.find(UUOne.class)
-      //.setDefaultLazyLoadBatchSize(5)
+    List<UUOne> list = DB.find(UUOne.class)
       .select("id")
       .where().startsWith("name", "testBLWCH")
       .order("name")
       .findList();
 
+    // invoke lazy loading
     for (UUOne uuOne : list) {
       uuOne.getName();
     }
     list.get(0).getName();
 
     List<String> sql = LoggedSqlCollector.stop();
-    System.out.println("sql:" + sql);
-
     assertThat(sql).hasSize(2);
     assertSql(sql.get(0)).contains("from uuone t0 where t0.name like ");
     platformAssertIn(sql.get(1), "from uuone t0 where t0.id");
 
     // not lazy loading into bean cache
-    int size = beanCache.getStatistics(true).getSize();
-    assertThat(size).isEqualTo(2);
```

Comment:
This wasn't right as previously we didn't hit the cache in the bulk lazy loading query and NOW we do hit L2 cache.